sympy/controlsys/lti.py

```diff
@@ -0,0 +1,1363 @@
+        """
+        result = []
+        for t_i in t_list:
+            # we use the arbitrary precision module mpmath for numerical evaluation of the matrix exponentials
+            first = mpm_matrix(self.represent[2].evalf()) * \
+                expm((self.represent[0] * (t_i - t0)).evalf()) * \
+                mpm_matrix(x0.evalf())
+
+            second = mpm_matrix((self.represent[3] * u.subs(t, t_i)).evalf())
+
+            def integrand(tau):
+                return \
+                    mpm_matrix(self.represent[2].evalf()) * \
+                    expm((self.represent[0] * (t_i - tau)).evalf()) * \
+                    mpm_matrix(self.represent[1].evalf()) * \
+                    mpm_matrix(u.subs(t, tau).evalf())
+
+            # the result must have the same number of rows as C:
+            integral = mpm_matrix(self.represent[2].rows, 1)
+
+            # Loop through every entry and evaluate the integral using mpmath.quad()
+            for row_idx in range(self.represent[2].rows):
+
+                    integral[row_idx] = quad(lambda x: integrand(x)[row_idx], [t0, t_i])
+
+            result.append(Matrix((first + second + integral).tolist()))
+
+        # return sum of results
+        return result
+
+    #
+    # _solve_symbolically
+    #
+    def _solve_symbolically(self, u, x0, t, t0, do_integrals):
+        """ returns the symbolic evaluation of the system for input u and known state x0 at time t0
+        """
+        # define temporary symbols tau
+        tau = Symbol('tau', positive=True)
+        x = Symbol('x')
+
+        # compute the two matrix exponentials that are used in the general solution
+        # to avoid two eigenvalue problems, first solve for a general real x and substitute then
+        expAx = exp(self.represent[0] * x)
+        expA = expAx.subs(x, t - t0)
+        expAt = expAx.subs(x, t - tau)
+
+        # define the integral and heuristic simplification knowing that in the integral, tau < t always holds
+        integrand = self.represent[2] * expAt * self.represent[1] * u.subs(t, tau)
+        integrand = integrand.subs([(abs(t - tau), t - tau), (abs(tau - t), t - tau)])
+        integral = zeros(integrand.shape[0], integrand.shape[1])
+        for col_idx in range(integrand.cols):
+
+            for row_idx in range(integrand.rows):
+                try:
+                    if not integrand[row_idx, col_idx] == 0:
+                        if do_integrals is True:
+                            integral[row_idx, col_idx] = \
+                                integrate(integrand[row_idx, col_idx], (tau, t0, t))
+                        else:
+                            integral[row_idx, col_idx] = \
+                                Integral(integrand[row_idx, col_idx], (tau, t0, t))
+                except:
+                    integral[row_idx, col_idx] = \
+                        Integral(integrand[row_idx, col_idx], (tau, t0, t))
```

Comment:
We should never allow bare except. Hopefully at some point we can get flake8 to automatically disallow things like this.